client/lib/browser-storage/index.ts

```diff
@@ -145,20 +148,20 @@ function idbGetAll( pattern?: RegExp ): Promise< StoredItems > {
 
 let idbWriteCount = 0;
 let idbWriteBlock: Promise< void > | null = null;
-function idbSet< T >( key: string, value: T ): Promise< void > {
+async function idbSet< T >( key: string, value: T ): Promise< void > {
```

Comment:
It looked odd to me that you were mixing async and promise syntax, but TIL that returning a promise from an async function is the same as returning the resolved value 🙂